src/main/java/seedu/address/model/calendarevent/DateTime.java

```diff
@@ -11,80 +11,106 @@
  */
 public class DateTime {
 
-    private LocalDateTime date;
+    public static final String MESSAGE_DATETIME_CONSTRAINTS =
+            "The valid input will be year-month-localDateTime hour:minute";
```

Comment:
should probably be "year-month-day hour:minute"

Reply:
you are right